README.md

```diff
@@ -39,18 +39,247 @@ It provides a **data & learning flywheel for LLMs** by unifying:
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/guides/providers/mistral">Mistral</a></b></li>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/guides/providers/openai">OpenAI</a></b></li>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/guides/providers/together">Together</a></b></li>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/guides/providers/vllm">vLLM</a></b></li>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/guides/providers/xai">xAI</a></b></li>
+      </ul>
+      <p>
+        <em>
+          Need something else?
+          Your provider is most likely supported because TensorZero integrates with <b><a href="https://www.tensorzero.com/docs/gateway/guides/providers/openai-compatible">any OpenAI-compatible API (e.g. Ollama)</a></b>.
+          </em>
+      </p>
+    </td>
+    <td width="50%" align="left" valign="top">
+      <p>
+        The TensorZero Gateway supports advanced features like:
+      </p>
+      <ul>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/guides/retries-fallbacks">Retries & Fallbacks</a></b></li>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/guides/inference-time-optimizations">Inference-Time Optimizations</a></b></li>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/guides/prompt-templates-schemas">Prompt Templates & Schemas</a></b></li>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/tutorial#experimentation">Experimentation (A/B Testing)</a></b></li>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/configuration-reference">Configuration-as-Code (GitOps)</a></b></li>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/guides/batch-inference">Batch Inference</a></b></li>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/guides/metrics-feedback">Metrics & Feedback</a></b></li>
+        <li><b><a href="https://www.tensorzero.com/docs/gateway/guides/episodes">Multi-Step LLM Workflows (Episodes)</a></b></li>
+        <li><em>& a lot more...</em></li>
+      </ul>
+      <p>
+        The TensorZero Gateway is written in Rust 🦀 with <b>performance</b> in mind (&lt;1ms p99 latency overhead @ 10k QPS).
+        See <b><a href="https://www.tensorzero.com/docs/gateway/benchmarks">Benchmarks</a></b>.<br>
+      </p>
+      <p>
+        You can run inference using the <b>TensorZero client</b> (recommended), the <b>OpenAI client</b>, or the <b>HTTP API</b>.
+      </p>
+    </td>
+  </tr>
+</table>
+
+<br>
+
+<details>
+<summary><b>Usage: TensorZero Python Client (Recommended)</b></summary>
+
+You can access any provider using the TensorZero Python client.
+
+1. Deploy `tensorzero/gateway` using Docker.
+   **[Detailed instructions →](https://www.tensorzero.com/docs/gateway/deployment)**
+2. Optional: Set up the TensorZero configuration.
```

Comment:
we should probably write a test that does zero-config tensorzero with clickhouse just to make sure it works

Reply:
can you open an issue?